### DISCUSSION TITLE
Modifying device names without 'rebooting'?

### DISCUSSION THREAD

**Discussion by EJansenMSFT** [jertel/vuegraf#49] [Q&A] [answered]
So lets say that I have the following circuits defined / monitored in the Emporia app:

![image](https://user-images.githubusercontent.com/72317762/120092286-c4fbac00-c0df-11eb-9176-b555c804fb88.png)

but then maybe i consolidated loads to another circuit and then moved the CT clamp to a different circuit to monitor, and now I want it to say this instead:

![image](https://user-images.githubusercontent.com/72317762/120092321-0ab87480-c0e0-11eb-8c78-e629a1065753.png)

How can I  change those values and re-load them into grafana, without restarting?

Thanks,
Eric

**Comment by EJansenMSFT** [comment, marked as answer]
Disregard, I just cycled the vuegraf container and it reloaded based on the data in the .json file.  My worry was data loss, but I suppose that would only be the case if I rebooted the InfluxDB container?  Still don't have a full grasp of how it all works just yet.

**Reply by jertel** [reply]
Docker-compose creates and maintains the volume for you, so restarting the influxdb container shouldn't lose your data:
![image](https://user-images.githubusercontent.com/16639611/120103281-e4fd9080-c11c-11eb-81f6-26b57eb0571c.png)

From https://docs.docker.com/storage/volumes/.

**Reply by EJansenMSFT** [reply]
Awesome, thanks for the detailed response, and great project btw!
